### PR TITLE
Generator - RandomTest

### PR DESCRIPTION
**Pull request by zhikezhang**
This PR creates the RandomTest. 
1. Add RandomTest, generating dapp automatically. This method has random number of arguments. The arguments of this method have randum types and random values. This dapp returns the abi-encoded arguments.
2. Also, Deploy, call and verify the randomly generated dapp.

### CODE REVIEW

File: test/IntegTest.java
Comment (arajasek) [inline test/IntegTest.java:161]: All of the changes to this file are redundant, so this file should not be included as part of the PR.

File: test/RandomTest.java
Comment (arajasek) [inline test/RandomTest.java]: Unused imports can be taken out.

File: test/RandomTest.java
Comment (arajasek) [inline test/RandomTest.java]: I would comment out this `println`, and the ones for the parameters. I think so long as you print out the seed, we should always be able to recreate the random test, which is all we want.

It gets irritating to have very verbose tests.

**Comment by zhike-aion** [comment]
Also, Re-organize the test folder, improve the code structures, etc..

File: test/RandomTest.java
Comment (arajasek) [inline test/RandomTest.java:156]: Please insert comments explaining what the generated method(s?) do.

**Review by arajasek — CHANGES_REQUESTED** [review]
Thanks for working on this! I have some initial requested changes, might have more on a second review.

File: test/RandomTest.java
Comment (zhikezhang) [reply test/RandomTest.java]: Good point!

File: test/RandomTest.java
Comment (zhikezhang) [reply test/RandomTest.java]: Good point!

File: test/IntegTest.java
Comment (zhikezhang) [reply test/IntegTest.java:161]: Good point!

**Review by arajasek — APPROVED** [review]
Thanks for the changes, looks good to me!

**Pull request merged**
